.github/workflows/check-disclosures.yml

```diff
@@ -0,0 +1,33 @@
+name: Check disclosures 
+
+on: 
+  pull_request:
```

Comment:
Following my comment here: https://github.com/418sec/huntr/pull/58#issuecomment-645410772

We can specify which actions trigger the action.